src/main.cpp

```diff
@@ -195,6 +197,10 @@ main(int argc, char *argv[])
         }
 
         QSettings settings;
```

Comment:
You can still delete this!

```suggestion
```

Reply:
Hmm, looks like somebody added some things to UserSettings but never used them in main...

Reply:
Actually, there are things in UserSettings to set font size & family and the start in tray values (what the QSettings is used for in main()), but not to get them... Looks like I've got a bit more work :P

Reply:
Well, I just don't like reading the settings file twice immediately in main :D

Reply:
OK, forget my previous comment